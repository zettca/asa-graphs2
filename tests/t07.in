30 50
22
27 15 -2
28 3 2
25 11 -3
9 19 0
22 18 -3
27 20 0
2 30 -2
23 14 -1
14 10 -1
18 9 -3
18 29 0
2 21 3
10 3 -3
20 16 -1
5 26 1
3 1 -3
8 9 1
1 11 -2
10 7 1
21 30 -2
3 6 -1
5 14 2
30 6 -1
17 4 -2
18 11 -2
24 12 0
30 7 3
21 13 1
28 25 0
16 17 -2
28 23 -3
8 2 3
13 27 1
12 3 -1
4 5 2
18 10 3
29 13 -1
8 14 3
5 30 0
6 17 3
23 16 0
2 13 -3
15 26 -1
19 22 1
27 7 0
25 2 2
26 1 0
2 14 2
26 14 -1
21 18 -3
