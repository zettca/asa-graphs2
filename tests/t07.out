I
U
I
I
I
I
I
U
I
I
I
U
I
I
I
I
I
I
I
I
U
I
U
U
U
I
I
U
I
I
